fbfmaproom/fbfmaproom.py

```diff
@@ -106,14 +105,60 @@ def units(self):
             self._units = self.open().attrs.get('units')
         return self._units
 
+    def open(self, val_min=None, val_max=None):
+        path = data_path(self.path)
+        try:
+            ds = xr.open_zarr(path, consolidated=False)
+        except Exception as e:
+            raise Exception(f"Couldn't open {path}") from e
+        ds = ds.rename({
+            v: k
+            for k, v in self.var_names.items()
+            if v is not None and v != k
+        })
+        da = ds["value"]
+
+
+        if val_min is None:
+            val_min = self.range[0]
+        if val_max is None:
+            val_max = self.range[1]
+        da.attrs["colormap"] = self.colormap
+        da.attrs["scale_min"] = val_min
+        da.attrs["scale_max"] = val_max
+        return da
+
 
 class ObsDataset(Dataset):
```

Comment:
Wasn't sure where to put this but do you think these classes or at least the base class belong in Pingrid? I feel like some of this functionality could be used in our other maprooms. Some is conceptually similar to stuff I've been doing. Perhaps this just needs to be revisited once my library is done (soon)

Reply:
Yeah maybe someday. Fleshing out the ideas here first.